pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -112,7 +112,7 @@ spec:
         - name: data
           workspace: release-workspace
       runAfter:
-        - verify-access-to-resources
+        - reduce-snapshot
```

Comment:
This is collect-data, which is saying run after reduce-snapshot. Shouldn't it be apply-mapping that runs after? Right now, collect-data runs after reduce-snapshot and reduce-snapshot runs after collect-data